src/Provider/AvailableShippingMethodsListProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Sylius\AdminOrderCreationPlugin\Provider;
+
+use Sylius\AdminOrderCreationPlugin\Entity\OrderInterface;
+use Sylius\Component\Shipping\Resolver\ShippingMethodsResolverInterface;
+
+final class AvailableShippingMethodsListProvider
+{
+    /** @var ShippingMethodsResolverInterface */
+    private $shippingMethodsResolver;
+
+    public function __construct(ShippingMethodsResolverInterface $shippingMethodsResolver)
+    {
+        $this->shippingMethodsResolver = $shippingMethodsResolver;
+    }
+
+    public function __invoke(OrderInterface $order, int $shipmentNumber): array
```

Comment:
`$shipmentNumber` is quite vague for me, what if we pass `ShipmentInterface` instance instead?